apps/core-ie/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
```

Comment:
This demo is partially done. It demonstrates the global CSS and CSS custom properties work in IE. A followup PR will have demo of the web components.